docs/development/contributing/index.md

```diff
@@ -149,9 +149,9 @@ The user guide is maintained in the `docs/user-guide` folder. Before creating a
 
 #### Building and serving the user guide
 
-The user guide is built using [MkDocs](https://www.mkdocs.org/). You install the dependencies for building the user guide by running `make requirements` in the root of the repo.
+The user guide is built using [MkDocs](https://www.mkdocs.org/). You install the dependencies for building the user guide by running `make requirements` and `make build-python` in the root of the repo.
```

Comment:
what's `make build-python`?

Reply:
No longer exists - it should just be `make build`.

Reply:
Changed now